Automattic/jetpack
# VideoPress: check source language length of the video chapter

This PR looks at the source language length when uploading a new track for the video. When it's longer than five characters shows a notice error and blocks the `save` button.

Fixes https://github.com/Automattic/jetpack/issues/27647

## Proposed changes:

* VideoPress: check source language length of the video chapters

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add/edit a video block instance
* Click on the tracks button of the block toolbar
* Start to upload a new track
* Confirm an error notice raises when the source language is longer than five characters
* Confirm the app disables the `Save` button when there is that error

<img width="373" alt="Screen Shot 2023-01-17 at 11 33 08" src="https://user-images.githubusercontent.com/77539/212888625-628c2e95-64a9-4317-b40e-e6dca07b61cb.png">
